rollup.config.js

```diff
@@ -0,0 +1,115 @@
+import alias from 'rollup-plugin-alias'
+import babel from 'rollup-plugin-babel'
+import commonjs from 'rollup-plugin-commonjs'
+import resolve from 'rollup-plugin-node-resolve'
+import peerDepsExternal from 'rollup-plugin-peer-deps-external'
+import replace from 'rollup-plugin-replace'
+import { terser } from 'rollup-plugin-terser'
+
+const path = require('path')
+const fs = require('fs')
+const { join } = require('path')
+const { readdirSync } = require('fs-extra')
```

Comment:
Any reason to mix es5 and es6 imports ? :p

Reply:
Fixed